content/v3/migration/migrations.md

```diff
@@ -6,6 +6,8 @@ title: Migrations
 
 {:toc}
 
+The Migrations API *is only available to [authenticated](https://developer.github.com/v3/#authentication) organization owners*. Non-organization owners will receive a `404` response if they try to access it.
```

Comment:
IO'm not sure `organization owner` is correct anymore? I think `organization admin` may be better but I'm not sure.